tests/e2e/PHPUnit_012-5/composer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
todo: to extract the split with PHPUnit 12.5 with a link to the explanation as to why in the README.md